3rdparty/python/black.lockfile

```diff
@@ -0,0 +1,119 @@
+# This lockfile was autogenerated by Pants. To regenerate, run:
```

Comment:
Absolutely not a must-do, but you may want to consider adding these lockfiles to .gitattributes (so that they can be skipped at PR

Reply:
That's a great idea